docs/guide/rolldown.md

```diff
@@ -104,11 +147,15 @@ We encourage you to try out `rolldown-vite` and contribute to its development th
 - Oxc minifier is used for JS minification by default (esbuild was used before)
 - Rolldown is used for bundling the config (esbuild was used before)
 
-### Detecting rolldown-vite
+### Detecting `rolldown-vite`
 
-You can detect by either
+::: warning
+In most cases, you don't need to detect whether your plugin runs with `rolldown-vite` or `vite` and you should aim for consistent behavior across both, without conditional branching.
```

Comment:
I wonder if it is "us" (the Vite / Rolldown team) that aims for consistent behavior rather than "you" (the plugin authors).

Reply:
I'd say both is correct.

But as it is the plugin author's guide, the idea is to remind the plugin authors that there might not be a need for checking the version.